src/style/app.less

```diff
@@ -205,6 +205,10 @@ button.ga-btn, .ol-control button {
 }
 
 .ga-draw-active {
+
+  #search-container {
+    display: none;
+  }
```

Comment:
Can you add ` #search-container` with other components list which are already hidden when draw is  active see [line 225](https://github.com/geoadmin/mf-geoadmin3/blob/gjn_search_draw/src/style/app.less#L225)